test.js

```diff
@@ -12,11 +12,8 @@ describe('config', function() {
     it('should be configured', function() {
 
         CE = ChatEngine.create({
-            publishKey: 'pub-c-c6303bb2-8bf8-4417-aac7-e83b52237ea6',
-            subscribeKey: 'sub-c-67db0e7a-50be-11e7-bf50-02ee2ddab7fe',
-        }, {
-            endpoint: 'http://localhost:3000/insecure',
-            globalChannel: 'test-channel'
+            publishKey: 'pub-c-01491c54-379f-4d4a-b20b-9a03c24447c7',
```

Comment:
I'm not gonna block you cause these keys are already in GH, but we should try to remove these or encrypt the values.